app/views/schedules/new.html.erb

```diff
@@ -4,8 +4,8 @@
 <div class="row">
   <div class="col-md-8 col-md-offset-2">
     <%= form_for(@schedule) do |f| %>
-        <%= render partial: 'fields', locals: {f: f, object: @schedule} %>
-        <%= button_tag 'Add', type: :submit, class: 'btn btn-sm btn-primary' %>
+        <%= render partial: 'fields', locals: {f: f, object: @schedule } %>
+        <%= button_tag 'Add Root Task', type: :submit, class: 'btn btn-sm btn-primary' %>
```

Comment:
Ah, I see. To see how to add a single Task as part of this page, check out what we do to add a ParticleDevice to a Rhizome. You'll probably end up making a new partial (which isn't a problem at all), but you can base it off the existing one and strip the Cocoon-related stuff out.